### DISCUSSION TITLE
Lesson 1: Video on page 16 - argument "name" is not expected

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#299] [Course: Foundations of Private Computation]
Hi :)

Interesting issue.

<img width="813" alt="Screenshot 2021-03-24 at 15 39 53" src="https://user-images.githubusercontent.com/57599753/112329148-3d488a00-8cb7-11eb-9c97-bb606a6ef4de.png">
<img width="813" alt="Screenshot 2021-03-24 at 15 40 04" src="https://user-images.githubusercontent.com/57599753/112329153-3de12080-8cb7-11eb-8388-30aead09b835.png">

**Comment by znreza** [comment]
Hi @gonzalo-munillag the syft API has been changed in the newer version where "name" parameter has been removed. In the newer release there will be a better replacement of this parameter with some kind of request verification functionality. Till then please just remove the "name" parameter and it will work.
